src/main/scala/performanceanalysis/TimeParser.scala

```diff
@@ -0,0 +1,35 @@
+package performanceanalysis
+
+import java.time.LocalTime
+
+import scala.util.Try
+import scala.util.matching.Regex.Match
+
+class TimeParser(shift: Int) {
```

Comment:
This seems to be very tightly coupled to the regex parsers, because of the groups. Maybe `parse` can accept a list of strings(?).